include/nncase/data/dataset.h

```diff
@@ -94,7 +94,7 @@ class NNCASE_API dataset
         std::optional<data_batch<T>> value_;
     };
 
-    dataset(const std::filesystem::path &path, std::function<bool(const std::filesystem::path &)> file_filter, xt::dynamic_shape<size_t> input_shape, float mean, float std);
+    dataset(const std::filesystem::path &path, std::function<bool(const std::filesystem::path &)> file_filter, xt::dynamic_shape<size_t> input_shape, std::string input_axis, float mean, float std);
```

Comment:
It may be more appropriate to use `layout` instead of `axis`.

Reply:
fixed